content/articles/hashing-in-data-structure/index.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /hashing-in-data-structure/
+title: Hashing in data structure
+description: In this article, we will explore how the hash function converts a given key to a smaller number and uses the small number as an index in a table called a hash table.
+author: anubhav-bansal
+date: 2021-06-04T00:00:00-16:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /hashing-in-data-structure/hero.jpg 
+    alt: hero image example image
+---
+### Hashing in Data Structure
+
+In hashing, the idea is to use a hash function that converts a given key to a smaller number and uses the small number as an index in a table called a hash table.
+
```

Comment:
Briefly introduce what is hashing in general and what it entails. Most readers of this articles are beginner level. It's always good to put that in mind when writing an article.